gryphon/execution/lib/config_helper.py

```diff
@@ -83,6 +83,20 @@ def parse_extra_strategy_args(args):
     return output
 
 
+def get_conf_filename_from_strategy_name(strategy_name):
+    conf_filename = None
+
+    if strategy_name[-4:] == '.pyx':
+        conf_filename = '%s.conf' % strategy_name[:-4]
+    elif strategy_name[-3:] == '.py':
+        conf_filename = '%s.conf' % strategy_name[:-3]
+    else:
```

Comment:
The content of the else clause here could be the initialization at the beginning of this function. Or we don't need the initialization...

Reply:
This is actually the result of two sem-deliberate style decisions, avoiding variable initialization inside 'if' blocks, and preferring only a single return statement. I agree it sometimes makes for more lines than otherwise.